glide.yaml

```diff
@@ -9,10 +9,9 @@ import:
 - package: github.com/spacemonkeygo/monotime
 - package: github.com/uber/uber-licence
 - package: github.com/uber-go/zap
```

Comment:
I think its a good idea to pin to ^1 to avoid breaking changes if they release 2.x.x